semgrep-core/src/fixing/Autofix.mli

```diff
@@ -0,0 +1,6 @@
+val render_fix :
```

Comment:
Please add a small comment in the mli about the goal of the module and the return type of the function.